README.md

```diff
@@ -1,24 +1,112 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-app
 
-Things you may want to cover:
+> You will create a Ruby on Rails application that allows the user to:
 
-* Ruby version
+> - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- Boostrap
+- Postgres SQL
+- Rspec
+- Capybara
 
-* Database creation
+## Live Demo (if available)
```

Comment:
- [ ] You have done well by trying to deploy your application using Heroku. However, the application was not properly deployed as you can see below 👇. Please try to fix the issue with your deployment 🙏 
![image](https://user-images.githubusercontent.com/46839548/195940089-270201da-ce9a-4031-b316-744143833f4c.png)

Reply:
hey.. i have tried everything to fix this but It doesn't work. i really don't know what to do about this